askama_derive/src/parser.rs

```diff
@@ -668,7 +670,29 @@ macro_rules! expr_prec_layer {
         }
     }
 }
-
+#[cfg(feature = "localization")]
+fn localize(i: &str) -> IResult<&str, Expr<'_>> {
+    let (tail, (_, _, message, attribute, args, _)) = tuple((
+        tag("localize"),
+        ws(tag("(")),
+        identifier,
```

Comment:
I would like it better if this was a string, otherwise `greeting` in `localize(greeting, name: name)` looks like a variable to me. Maybe we actually could make it a normal Expr, for the least amount of confusion?

Reply:
So its quoted now, localize(greeting, name: name) is now localize("greeting", name: name)